Source/WebCore/css/CSSFontFaceSet.h

```diff
@@ -143,7 +143,7 @@ class CSSFontFaceSet final : public RefCounted<CSSFontFaceSet>, public CSSFontFa
     Status m_status { Status::Loaded };
     WeakHashSet<FontModifiedObserver> m_fontModifiedObservers;
     WeakHashSet<FontEventClient> m_fontEventClients;
-    WeakPtr<CSSFontSelector> m_owningFontSelector;
+    const WeakPtr<CSSFontSelector> m_owningFontSelector;
```

Comment:
A constant weak pointer is kind of a funny idea. Even if you don’t reassign it, it can reassign itself at any time. What’s the goal of marking it constant?